### DISCUSSION TITLE
BackTesting FinR

### DISCUSSION THREAD

**Discussion by antoniosavoldi** [AI4Finance-Foundation/FinRL#194] [Q&A]
Hi,
I need assistance regarding FinRL backtesting. When I train a model, for instance A2C (Actor Critic), on 10 years DowJones index, the learning experience is limited to the training data set (10 years of historical series which are converted in indicators that are feed into the model). Let us suppose that DowJones, at the moment, have a different composition compared to the training period. Therefore, it might be possible that the model might act on out-of-samples assets (e.g. choose actions on asset which has not seen before). How is it possible to backtest on this specific situation (i.e. I have a model which should act on different assets which have nott seen before)? How can we guarantee that the model is sufficiently general to be applied on new, unpreviously seen assets?
Thanks

**Comment by algogp** [comment]
I think you would need as input a mapping table to specify the time variation of its composition
